Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM imandra/imandra-client-switch as fix-engine-build
 
 COPY --chown=opam:nogroup ./fix-engine.opam .
+COPY --chown=opam:nogroup ./vendor/imandra-ptime/imandra-ptime.opam .
+COPY --chown=opam:nogroup ./vendor/imandra-prelude/imandra-prelude.opam .
```

Comment:
Yep, you're right that this is to avoid busting the docker cache - if we copied the whole folder here, we'd have to wait for the `opam install` step in the build when any file changed, not just the `.opam` files.